corehq/apps/users/models.py

```diff
@@ -1591,8 +1591,11 @@ def can_access_any_exports(self, domain=None):
             if permission_slug in EXPORT_PERMISSIONS
         ])
 
-    def can_login_as(self, domain):
-        return self.has_permission(domain, 'edit_commcare_users') or self.has_permission(domain, 'limited_login_as')
+    def can_login_as(self, domain=None):
```

Comment:
Is there code that calls `user.can_login_as()` (with no arguments)? If not, I would remove the `=None` default value from this function, but still allow `None` to be passed in.

Reply:
There is not, I'll update.

Reply:
Added to https://github.com/dimagi/commcare-hq/pull/27580 for the sake of saving a travis build (even though tests are performing much better now).